fe/projects/AspNetCore/src/areas/Products/components/ProductDetail/ProductDetail.js

```diff
@@ -22,15 +25,17 @@ function ProductDetail(props) {
     const [productVariant, setProductVariant] = useState(null);
     const [canActiveModal, setCanActiveModal] = useState(true);
     const [showMore, setShowMore] = useState(false);
+    const [showMoreImg, setShowMoreImg] = useState(false);
```

Comment:
**suggestion:** give full function names, example: showMoreImages and setShowMoreImages